reporter/src/main/kotlin/reporters/freemarker/FreemarkerTemplateProcessor.kt

```diff
@@ -179,6 +182,18 @@ class FreemarkerTemplateProcessor(
         return outputFiles
     }
 
+    /**
+     * Return a map with wrapper beans for the specified [enumClasses], so that these enums can be referenced
+     * directly by templates.
+     * See https://freemarker.apache.org/docs/pgui_misc_beanwrapper.html#jdk_15_enums.
+     */
+    private fun makeEnumsAvailable(enumClasses: Collection<Class<*>>): Map<String, Any> {
```

Comment:
Did you keep the function name intentionally as-is? Because this function does not "*make* enums available" by itself (applying the config does that). So maybe indeed rename this to my proposed `enumModel()` (to resemble `dataModel`), or to something else that comes to your mind.

Reply:
Renamed the function to `enumModel()` as proposed. The function is now alone responsible for this model and uses the `ENUM_CLASSES` constant for this purpose, no longer expecting it as parameter.

Reply:
> no longer expecting it as parameter.

Hmm, personally I would have kept it as a parameter and used the constant for it, but ok with me.